samples/04-file-transfer/transfer-file/src/main/java/org/eclipse/dataspaceconnector/extensions/api/FileTransferExtension.java

```diff
@@ -61,28 +58,25 @@ public void initialize(ServiceExtensionContext context) {
         DataFlowController dataFlowController = new InlineDataFlowController(vault, context.getMonitor(), dataOperatorRegistry, dataAddressResolver);
         dataFlowMgr.register(dataFlowController);
 
-        var policy = savePolicies(context);
```

Comment:
Just a minor thing, but I think this should be adjusted in the sample's README as well.

Reply:
done. also renamed the method to `createPolicy`